app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -10,10 +10,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     afterRender () {
-      const flowId = this.flow.get('flowId');
-      const flowBegin = this.flow.get('flowBegin');
-
-      this.metrics.logFlowBegin(flowId, flowBegin);
+      this.logFlowEventOnce('begin');
```

Comment:
We still need them both.

* `flow-begin-mixin` is mixed in to the views where we want to emit a `flow.begin` event. So sign-in, sign-up and force-auth, essentially. All that mixin does is emit the event, although it also needs `flow-events-mixin` in order to work.

* `flow-events-mixin` just handles initialisation of the flow model. This is needed for the above, but it's also needed for other views, e.g. during password reset or sign-in confirmation, where we want to initialise the flow model from a resume token or after navigation has occurred. Mixing it in to those views means that flow events continue to be emitted with the correct flow id, but without a fresh `flow.begin` event being emitted.